spec/requests/comments_create_spec.rb

```diff
@@ -93,4 +93,75 @@ def comment_params(params = {})
       expect(response.parsed_body["error"]).to be_present
     end
   end
+
+  context "when there's already a notification for comment" do
+    around do |example|
+      Sidekiq::Testing.inline!(&example)
+    end
```

Comment:
we do have `https://github.com/thepracticaldev/dev.to/blob/master/spec/support/sidekiq_test_helpers.rb` which contains a few helpers. this way you can explicitly process only the jobs you're interested in, which should also make the test faster.

Reply:
I've tried `sidekiq_perform_enqueued_jobs`, and it makes the tests pass without my changes. Due to unknown circumstances, mentions aren't being created with `sidekiq_perform_enqueued_jobs`.